src/admin/user-groups/views/UserGroupEdit.tsx

```diff
@@ -0,0 +1,470 @@
+			}
+			ToastService.success(t('De Gebruikersgroep is opgeslagen'), false);
+		} catch (err) {
+			console.error(
+				new CustomError('Failed to save user group', err, {
+					userGroup,
+					initialUserGroup,
+				})
+			);
+			ToastService.danger(t('Het opslaan van de gebruikersgroep is mislukt'), false);
+		}
+		setIsSaving(false);
+	};
+
+	const getAllPermissionGroups = () => {
+		const permissionGroupIdsInUserGroup: number[] = compact(
+			(get(userGroup, 'permissionGroups', []) as Partial<PermissionGroup>[]).map(pg => pg.id)
+		);
+		return allPermissionGroups
+			.filter(p => {
+				// Don't show permissionGroups that are already linked to this user group
+				return !permissionGroupIdsInUserGroup.includes(p.id as number);
+			})
+			.map(pg => ({
+				label: pg.label || pg.description || '',
+				value: String(pg.id),
+			}));
+	};
+
+	const renderTableCell = (rowData: Permission, columnId: UserGroupOverviewTableCols) => {
+		switch (columnId) {
+			case 'label':
+			case 'description':
+				return rowData[columnId];
+
+			case 'created_at':
+			case 'updated_at':
+				return formatDate(rowData[columnId]);
+
+			case 'actions':
+				return (
+					<ButtonToolbar>
+						<Button
+							icon="delete"
+							onClick={() => openModal(rowData)}
+							size="small"
+							ariaLabel={t('Verwijder')}
+							title={t('Verwijder')}
+							type="tertiary"
+						/>
+					</ButtonToolbar>
+				);
+
+			default:
+				return rowData[columnId];
+		}
+	};
+
+	const renderEditPage = () => {
+		if (!userGroup) {
+			return;
+		}
+		return (
+			<>
+				<Container size="medium">
+					<Spacer margin="bottom-extra-large">
+						<Box backgroundColor="gray">
+							<Form>
+								<FormGroup label={t('Label')} error={formErrors.label}>
+									<TextInput
+										value={userGroup.label || ''}
+										onChange={newLabel =>
+											setUserGroup({
+												...userGroup,
+												label: newLabel,
+											})
+										}
+									/>
+								</FormGroup>
+								<FormGroup label={t('Beschrijving')} error={formErrors.description}>
+									<TextInput
+										value={userGroup.description || ''}
+										onChange={newDescription =>
+											setUserGroup({
+												...userGroup,
+												description: newDescription,
+											})
+										}
+									/>
+								</FormGroup>
+							</Form>
+						</Box>
+					</Spacer>
+				</Container>
+
+				<Panel>
+					<PanelHeader>
+						<BlockHeading type="h3">
+							Permissie groepen gelinked aan deze gebruikersgroep:
+						</BlockHeading>
+					</PanelHeader>
+					<PanelBody>
+						<Spacer margin="bottom-large">
+							<Form type="inline">
+								<FormGroup
+									label={t(
+										'Voeg een permissie groep toe aan deze gebruikersgroep: '
+									)}
+								>
+									<Select
+										options={getAllPermissionGroups()}
+										placeholder={t('Kies een permissie groep')}
+										value={selectedPermissionGroupId}
```

Comment:
Crasht hier: `Type 'string | null' is not assignable to type 'string | undefined'`

Reply:
Idem.

Reply:
fixed, mijn local components dependency had nog changes, waardoor het bij mij wel werkte